stage0/.cargo/config.toml

```diff
@@ -2,9 +2,9 @@
 target = "x86_64-unknown-none"
 
 [target.x86_64-unknown-none]
-rustflags = "-C relocation-model=static -C code-model=large"
+rustflags = "-C relocation-model=static -C code-model=large -C linker-plugin-lto"
```

Comment:
Is adding `lto = true` to `Cargo.toml` not enough? We have that for the release builds, but not for dev builds.

If the dev build size is the problem, we could also just bump up the binary size, as LTO makes it more difficult to debug things.

Reply:
It wasn't previously when I was fully parsing the ELF file, but it now works again with parsing only the header and the program headers. Reverted this change.